### PR TITLE
chore: fix "lint" script & add "lint-fix" script

### PR DESCRIPTION
**Pull request by zenflow**
Fixes:
- Windows compatibility
- cover more files (e.g. files in cypress dir & root dir)

### CODE REVIEW

**Review by limonte — APPROVED** [review]
Thank you @zenflow  👍

**Comment by limonte** [comment]
Are you maybe planning to handle #192? I also have it on my todo list, TBH I didn't like the quality of the code in the external PR.

**Comment by zenflow** [comment]
@limonte yeah, I found the logic hard to follow. I reviewed again yesterday and still do. I think it's inherently complicated, but also I think it could probably be improved. Would need to wrap my head around it first lol. 

It's on my todo list as well but not sure when I'll get to it, so go ahead if you find time before me. 

I made a PR to add tests, could use your review: https://github.com/omerman/sweetalert2-react-content/pull/1
